boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -191,15 +209,30 @@ openapi-generate:
 .PHONY: generate
 generate: op-generate go-generate openapi-generate
 
+
 .PHONY: go-build
 go-build: ## Build binary
 	# Force GOOS=linux as we may want to build containers in other *nix-like systems (ie darwin).
 	# This is temporary until a better container build method is developed
 	${GOENV} GOOS=linux go build ${GOBUILDFLAGS} -o ${BINFILE} ${MAINPACKAGE}
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.23
```

Comment:
Is this version supposed to be the latest GA version that's used in OSD/ROSA clusters? If yes, should we include it in the gap analysis process [SOP](https://github.com/openshift/ops-sop/blob/master/v4/howto/gap-analysis.asciidoc)?

Reply:
> Is this version supposed to be the latest GA version that's used in OSD/ROSA clusters?

I am not quite sure, this version is used by latest osdk.

>If yes, should we include process of updating boilerplate image with new version as per gap analysis [SOP](https://github.com/openshift/ops-sop/blob/master/v4/howto/gap-analysis.asciidoc)?

what do you mean by this?

Reply:
Well, I just meant to ask about the frequency when this value `ENVTEST_K8S_VERSION = 1.23` will be updated. Right now, 4.10 (latest GA version) uses Kubernetes 1.23. So if this value is to be updated when 4.11 is GA, we might want to just track the change as part of gap analysis process in evaluating new version as well. Not sure if possible to have it done from a separate branch of boilerplate than `master` branch. Nothing majorly concerning right now so feel free to skip my query too.